generators/dart.js

```diff
@@ -299,3 +300,5 @@ Blockly.Dart.getAdjusted = function(block, atId, opt_delta, opt_negate,
   }
   return at;
 };
+
+exports = Dart;
```

Comment:
I think you have done it the preferable way since (as I understand it) mostly importers will be calling methods on the `Dart` object, but:

We could alternatively use named exports:

```suggestion
exports.Dart = Dart;
```

Reply:
Yeah i went for default export because of how it's used in the block generator files.